tensorlayer/layers/normalization.py

```diff
@@ -304,6 +305,126 @@ def __init__(
         self._add_params(variables)
 
 
+class GroupNormLayer(Layer):
+    """The :class:`GroupNormLayer` layer is for Group Normalization.
+    See `tf.contrib.layers.group_norm <https://www.tensorflow.org/api_docs/python/tf/contrib/layers/group_norm>`__.
+
+    Parameters
+    -----------
+    prev_layer : :class:`Layer`
+        The previous layer.
+    act : activation function
+        The activation function of this layer.
+    others : _
```

Comment:
we should put `groups` and `epsilon` here

Reply:
It's my mistake. I'll change it. Thanks.